Packs/BitcoinAbuse/Integrations/BitcoinAbuse/BitcoinAbuse.py

```diff
@@ -242,19 +252,68 @@ def fetch_indicators(params: Dict, test_module: bool):
             try:
                 indicator_id = int(indicator['rawJSON']['id'])
                 if indicator_id not in indicators_ids:
-                    indicator['fields']['cryptocurrencyaddresstype'] = 'bitcoin'
+                    update_indicator_occurrences(indicator)
                     indicators_without_duplicates.append(indicator)
                     indicators_ids.add(indicator_id)
             except ValueError:
                 demisto.debug(f'The following indicator was found invalid and was skipped: {indicator}')
 
-    if not test_module:
-        # we submit the indicators in batches
-        for b in batch(indicators_without_duplicates, batch_size=2000):
-            demisto.createIndicators(b)  # type: ignore
-        demisto.setIntegrationContext({'have_fetched_first_time': True})
-    else:
-        demisto.results('ok')
+    for indicator in indicators_without_duplicates:
+        indicator_count = indicators_name_to_count_dict.get(indicator['value'])
+        indicator['fields']['count'] = indicator_count
+        indicator['fields']['cryptocurrencyaddresstype'] = 'bitcoin'
+
+    return indicators_without_duplicates
+
+
+def bitcoin_abuse_test_module_command(params: Dict):
+    """
+    Performs a basic GET request to check if the API is reachable and authentication is successful.
+
+    Args:
+        params (Dict): Client object to perform request.
+
+    Returns:
+        'ok' if the call to Bitcoin Abuse service was successful and command is test_module.
+    """
+    get_indicators(params)
+    demisto.results('ok')
+
+
+def bitcoin_abuse_fetch_indicators_command(params: Dict) -> None:
+    """
+    Wrapper for fetching indicators from Bitcoin Abuse service.
+    Args:
+        params (Dict): Demisto params.
+
+    Returns:
+
+    """
+    indicators = get_indicators(params)
+    for b in batch(indicators, batch_size=2000):
+        demisto.createIndicators(b)  # type: ignore
+    demisto.setIntegrationContext({'have_fetched_first_time': True})
+
+
+def bitcoin_abuse_get_indicators_command(params: Dict, args: Dict):
```

Comment:
```suggestion
def get_indicators_command(params: Dict, args: Dict):
```

Reply:
The reason i added the prefix bitcoin_abuse is because CSV Module API has function named get_indicators_command, and they will collide
@ShahafBenYakir

Reply:
👍